### DISCUSSION TITLE
是否计划支持多人语音识别

### DISCUSSION THREAD

**Discussion by liweigu** [PaddlePaddle/PaddleSpeech#800] [Ideas]
从训练数据看目前是针对单人的语音识别，有计划支持多人同时说话的吗？

**Comment by zh794390558** [comment]
多人同时说话的不单是语音识别的任务了，是组合任务。这块需要收集需求，需要开发些盲源分离的模型或者结合信号前端波束的方式做，同时这个问题也不是容易解决的。暂时看没有这方面的安排，如果期望有，可以做强烈的反馈，同时我们会根据实际情况来决定做哪种方案。

**Reply by zh794390558** [reply]
可以的话，可以聊下你对语音的需求和使用情况。
